src/compose/ComposeMenu.js

```diff
@@ -202,10 +206,20 @@ class ComposeMenu extends PureComponent<Props> {
           </View>
         </AnimatedComponent>
         {!expanded && (
-          <IconPlusCircle style={this.styles.expandButton} size={24} onPress={onExpandContract} />
+          <IconPlusCircle
+            style={this.styles.expandButton}
+            size={24}
+            onPress={onExpandContract}
+            accessibilityLabel="Attach button"
```

Comment:
Hmm, I think "Attach button" isn't precisely the description for this button. One of the buttons doesn't attach anything; it takes you to a PM narrow, after you've selected the participants.

I see we've got "Collapse menu" for the button that shows in the same position when the menu has been expanded. What do you think about "Expand menu" for this one, as its opposite?

Reply:
"Expand menu" is looking apt here :)